packages/terra-text/src/terra-dev-site/doc/example/TextVisuallyHidden.jsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import Text from 'terra-text';
 
 const TextVisuallyHidden = () => (
   <div>
     <p>
+      The `isVisuallyHidden` prop can be used to hide the content visually, yet provide the content to screen readers.
```

Comment:
Lets move this description in the example render, not put it in the code: https://github.com/cerner/terra-core/pull/2886/files#diff-d549d5ca0552a55cbe5a1433f3ce80e6R58

Reply:
Updated here https://github.com/cerner/terra-core/pull/2886/commits/88d7739a51d61d65cec97299064fe301fcee4ce5.